src/test/java/com/revature/entities/ContentTest.java

```diff
@@ -131,7 +130,7 @@ public void testGetLinks() {
 
 	@Test
 	public void testSetLinks() {
-		Set<Link> link = new HashSet<Link>();
+//		Set<Link> link = new HashSet<Link>();
```

Comment:
remove > commenting out - does this test need to be here?

Reply:
I commented it out because I was not sure if I was allowed to remove old content at the time. I will remove it right now.

Reply:
Feel free to remove things if they're not needed 👍